pyscriptjs/tests/integration/test_zz_examples.py

```diff
@@ -190,7 +274,6 @@ def test_repl(self):
         )
         assert second_repl_result.text_content() == "4"
 
-    @pytest.mark.xfail(reason="Test seems flaky")
```

Comment:
Is this not flaky anymore? reason?

Reply:
Yeah not flaky anymore, the issue was resolved (or patched up) in #747  I ran `--count=20` just to confirm that it passed all the time and the test was always green 😄

Reply:
Great, thanks!